test/rauc.t

```diff
@@ -307,6 +307,19 @@ test_expect_success PKCS11 "rauc bundle with PKCS11 (key mismatch)" "
     bundle $SHARNESS_TEST_DIRECTORY/install-content ${TEST_TMPDIR}/out.raucb
 "
 
+test_expect_success ROOT "rauc mount" "
+  test ! -f /mnt/rauc/bundle/manifest.raucm &&
+  cp -L ${SHARNESS_TEST_DIRECTORY}/good-bundle.raucb ${TEST_TMPDIR}/ &&
+  test_when_finished rm -f ${TEST_TMPDIR}/good-bundle.raucb &&
+  test_when_finished umount /mnt/rauc/bundle &&
+  ls ${TEST_TMPDIR} &&
+  rauc --keyring $SHARNESS_TEST_DIRECTORY/openssl-ca/dev-ca.pem \
+    mount ${TEST_TMPDIR}/good-bundle.raucb &&
```

Comment:
Do you need to unmount this or is the test scaffolding responsible for that?

Reply:
That is handled by the ``test_when_finished umount /mnt/rauc/bundle`` above.